README.md

```diff
@@ -1,24 +1,47 @@
 # README
 
+## summary
+
 This README would normally document whatever steps are necessary to get the
 application up and running.
 
-Things you may want to cover:
+Things we may want to cover:
+
+- [ ] Ruby version
+
+- [ ] System dependencies
+
+- [ ] Configuration
+
+- [ ] Database creation
+
+- [ ] Database initialization
+
+- [ ] How to run the test suite
+
+- [ ] Services (job queues, cache servers, search engines, etc.)
 
-* Ruby version
+- [ ] Deployment instructions
 
-* System dependencies
+- [ ] ...
 
-* Configuration
+## Git workflow
+In the following subsections, the process of working with Git is described. (Contact @boomzilla if you have writing/stylistic suggestions.) (I am aiming for an audience fairly new to Git/Github, so if I my writing is unclear/overly complicated, please tell me. Although the converse isn't as much of a concern, feel free to mention it if I am needlessly belaboring a point.)
```

Comment:
Your comment to contact with suggestions might be good for a contributing section.